### PR TITLE
Results: figure 3 (gene expression vs. DNA methylation)

### PR DESCRIPTION
**Pull request by jjc2718**
Adding figure 3, comparing gene expression to DNA methylation, and text describing results.

image

### CODE REVIEW

**Comment by jjc2718** [comment]
Marking this as a draft for now - I'll almost certainly have to rewrite most of this based on the results with larger hyperparameter ranges.

File: content/04.results.md [hunk 1]
```diff
@@ -46,3 +46,38 @@ These results suggest that selecting target genes for mutation prediction based
...
 **C, D, E.** Volcano-like plots showing mutation presence/absence predictive performance for each gene in each of the 3 gene sets. The _x_-axis shows the difference in mean AUPR compared with a baseline model trained on shuffled labels, and the _y_-axis shows _p_-values for an unpaired _t_-test comparing cross-validated AUPR values.
 ](images/figure_2.png){#fig:expression_gene_sets width="90%"}
+
+### Comparing gene expression and DNA methylation reveals widespread redundancy
+
+We compared gene expression with DNA methylation as downstream readouts of the effects of cancer alterations.
+In these analyses, we considered both the 27K probe and 450K probe methylation datasets generated for the TCGA Pan-Cancer Atlas.
+Based on the results of the gene set experiments, we considered only the set of cancer-related genes derived from Vogelstein et al.
```
Comment (cgreene) [inline content/04.results.md:54]:
```suggestion
We performed this comparison using the cancer-related gene set derived from Vogelstein et al. [ref].
```

File: content/04.results.md [hunk 2]
```diff
@@ -46,3 +46,38 @@ These results suggest that selecting target genes for mutation prediction based
...
+We used samples that had data for each of the data types being compared, including somatic mutation data to generate mutation labels.
+This process retained 7,981 samples in the intersection of the expression, 27K methylation, 450K methylation, and mutation datasets, which we used for subsequent analyses.
+The most frequent missing data types were somatic mutation data (1,114 samples) and 450K methylation data (1,072 samples) (Figure {@fig:methylation}A).
+
+Overall performance across data types for 84 genes from is summarized in Figure {@fig:methylation}B.
```
Comment (cgreene) [inline content/04.results.md:59]:
```suggestion
Overall performance across data types for 84 genes is summarized in Figure {@fig:methylation}B.
```
I'm not sure you need this line. I think you are better off deleting it and referring to this figure in the line below.

**Review by cgreene — APPROVED** [review]
👍  Just a couple suggestions to help your readers

**Pull request merged**